tar/multitape/multitape.h

```diff
@@ -205,6 +205,14 @@ int statstape_close(TAPE_S *);
 int fscktape(uint64_t, const char *, int, int);
 
 /**
+ * initialize_cachedir(machinenum, cachedir):
+ * Initialize an empty chunk directory in ${cachedir} so that --print-stats
+ * works.  This requires the "directory" file, but no other files.  Return 0
+ * on success, -1 on error, and 1 if the cachedir is already initialized.
+ */
+int initialize_cachedir(uint64_t, const char *);
```

Comment:
This seems like an odd name as far as multitape function goes.  Something like `statstape_initialize` might make more sense?